HealthVault/Classes/Clients/MHVPersonClientProtocol.h

```diff
@@ -18,38 +18,72 @@
 
+/**
+ * Gets the application settings for the current application.
+ * The appSettingsXml will also be set on the MHVPersonInfo in getPersonInfoWithCompletion
+ *
+ * @param completion Envoked when the operation completes.
+ *        MHVApplicationSettings application settings.
+ *        NSError object will be nil if there is no error when performing the operation.
+ */
 - (void)getApplicationSettingsWithCompletion:(void(^)(MHVApplicationSettings *_Nullable settings, NSError *_Nullable error))completion;
 
-- (void)setApplicationSettingsWithRequestParameters:(NSString *)requestParameters
-                                         completion:(void(^_Nullable)(NSError *_Nullable error))completion;
+/**
+ * Sets the application settings for the current application.
+ * The applicationSettings.appSettingsXml will not be processed by HealthVault, but must be valid XML
+ *
+ * @param completion Envoked when the operation completes.
+ *        NSError object will be nil if there is no error when performing the operation.
+ */
+- (void)setApplicationSettings:(MHVApplicationSettings *)settings
+                    completion:(void(^_Nullable)(NSError *_Nullable error))completion;
 
 /**
  * Gets information about people authorized for an application.
  *
- * @param completion Envoked when the operation completes. NSArray<MHVPersonInfo *> an array of MHVPersonInfo objects representing people authorized for the application. NSError object will be nil if there is no error when performing the operation.
+ * @param completion Envoked when the operation completes.
+ *        NSArray of MHVPersonInfo objects representing people authorized for the application.
+ *        NSError object will be nil if there is no error when performing the operation.
  */
 - (void)getAuthorizedPeopleWithCompletion:(void(^)(NSArray<MHVPersonInfo *> *_Nullable personInfos, NSError *_Nullable error))completion;
 
-
 /**
  * Gets information about people authorized for an application.
  *
  * @param settings The MHVGetAuthorizedPeopleSettings object used to configure the results returned by this method.
- * @param completion Envoked when the operation completes. NSArray<MHVPersonInfo *> an array of MHVPersonInfo objects representing people authorized for the application. NSError object will be nil if there is no error when performing the operation.
+ *        This allows continuing if a previous getAuthorizedPeople call had the moreResults flag set
+ * @param completion Envoked when the operation completes. 
+ *        MHVGetAuthorizedPeopleResult containing an array of MHVPersonInfo objects representing people authorized for the application, and a moreResults flag.
+ *        NSError object will be nil if there is no error when performing the operation.
  */
-- (void)getAuthorizedPeopleWithSettings:(MHVGetAuthorizedPeopleSettings *_Nonnull)settings
+- (void)getAuthorizedPeopleWithSettings:(MHVGetAuthorizedPeopleSettings *)settings
```

Comment:
I think we should break out the authorizationsCreatedSince property on MHVGetAuthorizedPeopleSettings and only use that argument in this method call. MHVGetAuthorizedPeopleSettings seems like an internal object to me because we are handling fetching more results if the result is incomplete. In other words, the caller has no use for batchSize or startingPersonId.